### PR TITLE
improving imported-validator-status

### PR DESCRIPTION
**Pull request by gbayasgalan**
fix: https://github.com/stereum-dev/ethereum-node/issues/1801

### CODE REVIEW

File: launcher/src/components/UI/staking-page/components/list/rows/KeyRow.vue [hunk 1]
```diff
@@ -202,6 +212,46 @@ const getKeyState = computed(() => {
   }
 });
 
+const getKeyHanlingTime = computed(() => {
```
Comment (eiabea) [inline launcher/src/components/UI/staking-page/components/list/rows/KeyRow.vue:215]: Not quite sure if this is a typo? :thinking: 

```suggestion
const getKeyHandlingTime = computed(() => {
```

Comment (mabasian) [reply launcher/src/components/UI/staking-page/components/list/rows/KeyRow.vue:215]: nice catch!

File: launcher/src/components/UI/staking-page/components/list/rows/KeyRow.vue [hunk 2]
```diff
@@ -202,6 +213,72 @@ const getKeyState = computed(() => {
...
+    case "withdrawal":
+      return props.item.withdrawableSince;
+    case "NA":
+    case "loading":
+    default:
+      return "-";
+  }
+});
+
+const getKeyHanlingFooter = computed(() => {
```
Comment (eiabea) [inline launcher/src/components/UI/staking-page/components/list/rows/KeyRow.vue:237]: This is probably the same typo i already pointed out

```suggestion
const getKeyHandlingFooter = computed(() => {
```

**Review by mabasian — APPROVED** [review]
LGTM 🚀

**Pull request merged**